src/core/operations/FromBase85.mjs

```diff
@@ -33,6 +33,23 @@ class FromBase85 extends Operation {
                 value: ALPHABET_OPTIONS
             },
         ];
+        this.checks = [
+            {
+                pattern: "^\\s*(?:<~)?(?:(?:\\s*[!-u]){5}|\\s*z)+[!-u\\s]*(?:~>)?\\s*$",
+                flags: "i",
+                args: ["!-u"]
```

Comment:
Hello, thank you very much for contributing to the magic operation. 

The issue with base85 is that the alphabet spans such a large range, !-u, which includes all the upper case letters, most of the lower case letters and a bunch of punctuation. Thankfully, what it does not include is spaces, we need to modify these regexes so that they only match on inputs which are a certain length. The length should probably be in the region of 15-20, longer than most words. So this regex might become something like:

`^(<~)?(?:([!-u]|z)){20}[!-u]*(~>)?$`

Reply:
Well, the Ascii85 i.e. the Adobe version which seems to be the most common allows any whitespace anywhere and it should get ignored.

But I guess it makes sense to avoid spaces in the detection and search for larger continuous blocks. But then it should probably not use `^` and `$`. I guess it also makes sense to ignore the rule that `z` can only appear between groups of 5 other letters. I also just realized the `i` flag doesn't really make sense.

Would something like this be ok? `(?:<~)?\s*(?:[!-uz]){20,}\s*(?:~>)?`

Reply:
Actually, thinking about it there is no reason to include the `<~` since it won't change the result and I don't think it will improve performance. Just this should be fine: `(?:[!-uz]){20}`

Reply:
Or maybe a better option would be to require at least one block of 20 characters but still allow whitespace everywhere else and enforce that the whole input is valid: `^\s*(?:<~)?[\s!-uz]*[!-uz]{20}[\s!-uz]*(?:~>)?\s*$`.

Reply:
I'd say it is best to include the delimiter (<~) just to add to the accuracy of the regex. 

Including a space in an example base85 encoded string breaks the fromBase85 operation, which means it would then not be seen as successful by magic. So without further modification to the fromBase85 operation then we shouldn't include it in the regex.

Reply:
> I also made FromBase85 ignore whitespace since Ascii85 is supposed to ignore all whitespace according to Wikipedia (History > Adobe version).

I added whitespace support in this PR already.

Including `<~` makes sense but in a regex like this `(?:<~)?\s*[!-uz]{20,}\s*(?:~>)?` it will not make a difference. Since it is optional it only matters when there is something before or after it, like the `^`.

I think this (`^\s*(?:<~)?[\s!-uz]*[!-uz]{20}[\s!-uz]*(?:~>)?\s*$`) should be fine. It will reject everything that doesn't have at least 20 characters of continuous base85 but it still allows arbitrary whitespace otherwise and it also enforces that the whole input doesn't contain any non-base85 characters. It might still match some long uppercase sequences but I think that's unavoidable and I think most base85 strings will at least have some continuous parts or more likely not whitespace at all.